docs/PIPELINES-CONFIG.md

```diff
@@ -0,0 +1,269 @@
+## Pipelines/Tekton Configuration
+
+As part of this section you will find information about the following configuration methods:
+
+1. [Configuring Pipelines/Tekton with the configuration script after using our `ai-rhdh-installer`](#ai-rhdh-installer-script-configuration)
+2. [Configuring Pipelines/Tekton manually after using our `ai-rhdh-installer`](#ai-rhdh-installer-manual-configuration)
+3. [Configuring Pipelines/Tekton with the configuration script for a pre-existing Red Hat Developer Hub instance](#pre-existing-instance-script-configuration)
+4. [Configuring Pipelines/Tekton manually for a pre-existing Red Hat Developer Hub instance](#pre-existing-instance-manual-configuration)
+
+### Prerequisites
+
+- [yq](https://github.com/mikefarah/yq/) version 4.0+.
+- [kubectl](https://github.com/kubernetes/kubectl) version compatible with your target cluster
+
+### AI-RHDH-Installer: Script Configuration
+If you installed using the [`ai-rhdh-installer`](../README.md#install) all that is required for hooking up the Pipelines services and Tekton to your RHDH instance is to run `bash ./scripts/configure-tekton.sh`. Please note if you changed the installation namespace used by the installer you will first need to run `export NAMESPACE=<namespace used>` as the default value is `ai-rhdh`.
+
+### AI-RHDH-Installer: Manual Configuration
+
+#### Step 1: TektonConfig CR
+You should patch the `TektonConfig` CR called `config` with the extended configuration under [`tekton-config.json`](../resources/tekton-config.json)
```

Comment:
might be good to include an example command here?

Reply:
> might be good to include an example command here?

Yeah I think I missed this part.

Reply:
Fixed in https://github.com/redhat-ai-dev/ai-rhdh-installer/pull/7/commits/8b46981fcee936554bd5dbb95cd294caa83734d8